blog/arquillian-pact-1.0.0.Alpha2.textile

```diff
@@ -0,0 +1,73 @@
+---
+date: 2016-09-29
+author: lordofthejars
+---
+
+p. Arquillian Pact is an extension that provides an integration between Arquillian and "Pact":https://docs.pact.io/.
+
+h3 What's new
+
+p. <i class="icon-star"></i> Maven Loader
+
+p. Now you can use any Maven artifact to store contracts (pacts files). Then provider will resolve given artifact, uncompress it and execute the contracts found there.
+
+p. To use it you need to import dependency:
+
+bc(prettify).. <dependency>
+  <groupId>org.arquillian.pact</groupId>
+  <artifactId>arquillian-pact-provider-maven-loader</artifactId>
+</dependency>
+
+p. For example, this is how you can get contracts from a jar file stored at Maven repository. It is important to note that Arquillian Pact always takes the latest version of jar in case of not specifying one concrete version.
```

Comment:
> latest version of jar in case of not specifying one concrete version.

"latest version of **a** jar, when no concrete version is specified." ?

Question: latest what? release? snapshot?